AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1079,7 +1082,14 @@ - (void)__layout
     // measureWithSizeRange: on subnodes to assert.
     return;
   }
+  
+  // Handle placeholder layer creation in case the size of the node changed after the initial placeholder layer
+  // was created
+  if ([self _shouldHavePlaceholderLayer]) {
```

Comment:
@maicki This change is unrelated to the rest of the PR, right?  I recall us committing something like this recently, but was reverted due to another bug?

Anyway, if it's intended to be here - can we add to the comment a brief description of why _setup is helpful in the size change case?  It's not totally clear to me, because we're about to set the .frame below anyway...so why does it matter if the node size changed / what side effect does _setup have that makes this necessary?  Thanks much!